src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
+
+def install_debuginfos():
+    """Install debuginfo packages"""
+
+    install_pkgs = [{'name': 'gdb'}, {'name': 'python-magic'}]
+    cmds = []
+
+    if USE_DEBUGINFO_INSTALL:
+        cmds.append("sudo debuginfo-install -y "                   \
+                    "--exclude ompi-debuginfo,gcc-debuginfo,"      \
+                               "gcc-base-debuginfo "               \
+                    "daos-server cart libpmemobj python openmpi3")
+    else:
+        import yum
```

Comment:
(pylint-import-error) Unable to import 'yum'

Reply:
I don't know how to resolve this.  This python module is contained in the base `yum` RPM.

Reply:
Typically we need to install the python3 version of the yum module for it to be detected by the python3 version of pylint that we now run.

Reply:
Ah right.  We really need to get off of this dual python version bandwagon.  But then again, that would actually cripple us due the lagginess of python3 on EL7.

In any case, there is no python3 version of this module on EL7.

Reply:
Have you tried adding `pip3 install yum` to the `Dockerfile.centos.7`?